web/styles/user_status.css

```diff
@@ -40,15 +41,11 @@
             /* For custom emojis and smiley icon to take full width. */
             & img.selected-emoji,
             .smiley-icon {
-                min-width: 20px;
+                min-width: 16px;
```

Comment:
Can you talk a little here about why this width change is needed?

Reply:
| Before | After |
| --- | --- |
|![Screenshot from 2023-12-20 22-38-06](https://github.com/zulip/zulip/assets/90370535/1d96a961-fd8d-465b-9931-b4da329875d1) |![Screenshot from 2023-12-20 22-38-19](https://github.com/zulip/zulip/assets/90370535/40dc59bf-bd17-4020-866c-9b1e8065c1af)

this is to fix extra margin from right side.

Reply:
I have reverted the min-width back to 20px to ensure smooth functioning of custom emojis. I have used text align center now for horizontal alignment.